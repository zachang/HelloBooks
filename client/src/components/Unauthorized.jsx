import React from 'react';

/**
 * Unauthorized class declaration
 *
 * @class Unauthorized
 *
 * @extends {React.Component}
 */
export default class Unauthorized extends React.Component {
  /**
   * Renders Unauthorized component
   *
   * @return {XML} JSX
   */
  render() {
    return (
      <div>
        <div className='unauthorized'>
        </div>
      </div>
    );
  }
}
