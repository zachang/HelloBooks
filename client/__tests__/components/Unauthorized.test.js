import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Unauthorized from '../../src/components/Unauthorized.jsx';

describe('<Unauthorized/>', () => {
  const shallowWrapper = shallow(<Unauthorized/>);
  const tree = toJson(shallowWrapper);


  it('renders <Unauthorized/> component', () => {
    expect(tree).toMatchSnapshot();
  });
});