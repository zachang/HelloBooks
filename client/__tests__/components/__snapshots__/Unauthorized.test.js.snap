// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Unauthorized/> renders <Unauthorized/> component 1`] = `
<div>
  <div
    className="unauthorized"
  />
</div>
`;
